docs/guide/multisite/siteaccess_aware_configuration.md

```diff
@@ -0,0 +1,251 @@
+# SiteAccess-aware configuration
+
+The [Symfony Config component](https://symfony.com/doc/5.0/components/config.html) makes it possible to define semantic configuration, exposed to the end developer.
+This configuration is validated by rules you define, e.g. validating type (string, array, integer, boolean, etc.).
+Usually, once validated and processed, this semantic configuration is then mapped to internal *key/value* parameters stored in the service container.
+
+[[= product_name =]] uses this for its core configuration, but adds another configuration level, the SiteAccess.
+For each defined SiteAccess, you need to be able to use the same configuration tree to define SiteAccess-specific config.
+
+These settings then need to be mapped to SiteAccess-aware internal parameters 
+that you can retrieve with the [ConfigResolver](../config_dynamic.md#configresolver).
+For this, internal keys need to follow the format `<namespace>.<scope>.<parameter_name>`. where:
+
+- `namespace` is specific to your app or bundle
+- `scope` is the SiteAccess, SiteAccess group, `default` or `global`
+- `parameter_name` is the actual setting *identifier*
+
+For more information about the ConfigResolver, namespaces and scopes, see [[[= product_name =]] configuration basics](../configuration.md).
+
+### Semantic configuration parsing
+
+To parse semantic configuration, create a `Configuration` class which extends
+`eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\Configuration`
+and the extend its `generateScopeBaseNode()` method:
```

Comment:
Not sure where to place this remark, so I'll put it here. While this is correct, it's not nearly enough. There's an implicit assumption of the existence of AppBundle which comes from Symfony 2- 3 boilerplate code. This is no longer the case for Symfony 5+ - bundles are gone. On the other hand Bundle is required to load configuration extension, so need to be added.
One of the possibilities of full configuration is shown is [this diff](https://github.com/alongosz/ibexa-open-source/compare/564a5cf..e9887ab)

Reply:
```suggestion
To parse semantic configuration, create a `Configuration` class which extends
`eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\Configuration`
and extend its `generateScopeBaseNode()` method:
```

Reply:
```suggestion
To parse semantic configuration, create a `Configuration` class which extends
`eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\Configuration`
and then extend its `generateScopeBaseNode()` method:
```